### ISSUE TITLE
Pi.Alert failing to get internet IP when IPv6 is used

### ISSUE THREAD

**Issue by roberthawdon** [pucherot/Pi.Alert#12]
When running `python ~/pialert/back/pialert.py internet_IP` I get the following output:

```
Pi.Alert 2.52 (2021-01-11)
---------------------------------------------------------
Check Internet IP
    Timestamp: 2021-01-13 05:53:00

Retrieving Internet IP...
    Error retrieving Internet IP
    Exiting...
```

After examining the code, it appears that Pi.Alert has no way of handling an IPv6 response from opendns. I also couldn't find a way to make opendns respond with my IPv4 address, so to workaround the issue for now, I changed the lookup service to `ipv4.icanhazip.com` by changing line 160 in pialert.py to read:

```curl_args = ['curl', '-s', 'http://ipv4.icanhazip.com']```

Will Pi.Alert be updated to check both IPv4 and IPv6 addresses?

**Comment by pucherot** [comment]
Hi Robert
please, can you execute this commands and paste the result?
~~~
curl -s https://diagnostic.opendns.com/myip
curl -s http://ipv4.icanhazip.com
~~~
(if you want, you can replace some characters with #)

**Comment by kotahukka** [comment]
not Robert but I'm having the same issue.
when I run the above commands I get 
curl -s https://diagnostic.opendns.com/myip
2600:1700:12e2:###::###
curl -s http://ipv4.icanhazip.com
75.57.12#.#

**Comment by roberthawdon** [comment]
Hi, apoligies for the delay:

`curl -s https://diagnostic.opendns.com/myip` returns:
`2a02:c7f:98c2:####:####:####:####:####`

`curl -s http://ipv4.icanhazip.com` returns:
`151.231.###.###`

**Comment by pucherot** [comment]
Solved in v2.55.
Now default Server to query you IP es http://ipv4.icanhazip.com
You can configure it in pyalert.conf

**Comment by pucherot** [comment]
Hello again

Please, can you test this command
`dig @resolver4.opendns.com myip.opendns.com +short`
